app/assets/javascripts/linkers.js.coffee

```diff
@@ -1,3 +1,24 @@
-# Place all the behaviors and hooks related to the matching controller here.
-# All this logic will automatically be available in application.js.
-# You can use CoffeeScript in this file: http://coffeescript.org/
+$ ->
+  $('#test-button').click (e) ->
+    e.preventDefault()
+    $('#test-results').show()
+    readCSVFile()
+
+  readCSVFile = ->
+    csvContents = $('#CSVfile')[0].files[0]
+    reader = new FileReader()
+
+    reader.onload = (event) ->
+      reader.result.split('\n').forEach (line) ->
+        $tr = $('<tr>')
+        $tr.append $('<td>' + line + '</td>')
+
+        requestParams =
+          url_string: line
+          search_term: $('#search-term').val()
+
+        $.getJSON('/ping_url', requestParams).done (result) ->
```

Comment:
[According to SO](http://stackoverflow.com/a/15055915/974981), Rails helpers aren't available in CoffeeScript ERB files :(